packages/components/src/Offer/Form/Bid.tsx

```diff
@@ -243,9 +243,7 @@ const OfferFormBid: FC<Props> = (props) => {
         <InputGroup>
           <NumberInput
             clampValueOnBlur={false}
-            min={0}
```

Comment:
I think we should keep the `min={0}` property to not let the stepper goes negative.

Reply:
Added min and max where needed to prevent stepper to go below/above